packages/tenant-process/src/services/tenantService.ts

```diff
@@ -138,6 +205,9 @@ export function tenantServiceBuilder(
         tenantSeed.externalId
       );
```

Comment:
I suggest to add also a log before the retrieve, because in case of error during the query we would not have info about the `externalId` (for privacy reasons the request payload is not logged automatically)